tt_eager/tt_dnn/op_library/moreh_norm/moreh_norm_h/kernels/moreh_norm_h_kernel.cpp

```diff
@@ -80,60 +80,68 @@ void MAIN {
 
             abs_tile_init();
             abs_tile(dst0);
+			tile_regs_commit();
 
-            pack_tile(dst0, cb_xabs);
+			tile_regs_wait();
+            pack_tile_with_dt(dst0, cb_xabs);
+			tile_regs_release();
 
             cb_pop_front(cb_x, onetile);
             cb_push_back(cb_xabs, onetile);
-            REL();
 
             power_tile_to_cb(cb_xabs, cb_xpow, cb_logx, cb_decimal, cb_exp_lxmd, cb_correct_xpow, p, p_is_negative);
 
             // Add(|x|^p)
             if (row_idx == 0) {
-                ACQ();
+				tile_regs_acquire();
                 cb_wait_front(cb_correct_xpow, onetile);
                 cb_reserve_back(cb_xpowadd, onetile);
 
-                copy_tile_init();
+                copy_tile_init_with_dt(cb_correct_xpow);
                 copy_tile(cb_correct_xpow, 0, dst0);
+				tile_regs_commit();
 
-                pack_tile(dst0, cb_xpowadd);
+				tile_regs_wait();
+                pack_tile_with_dt(dst0, cb_xpowadd);
+				tile_regs_release();
 
                 cb_pop_front(cb_correct_xpow, onetile);
                 cb_push_back(cb_xpowadd, onetile);
-                REL();
             } else {
-                ACQ();
+				tile_regs_acquire();
                 cb_wait_front(cb_correct_xpow, onetile);
                 cb_wait_front(cb_xpowadd, onetile);
                 cb_reserve_back(cb_xpowadd, onetile);
 
-                add_tiles_init();
+                add_tiles_init_with_dt(cb_correct_xpow, cb_xpowadd);
                 add_tiles(cb_correct_xpow, cb_xpowadd, 0, 0, dst0);
+				tile_regs_commit();
 
-                pack_tile(dst0, cb_xpowadd);
+				tile_regs_wait();
+                pack_tile_with_dt(dst0, cb_xpowadd);
+				tile_regs_release();
 
                 cb_pop_front(cb_correct_xpow, onetile);
                 cb_pop_front(cb_xpowadd, onetile);
                 cb_push_back(cb_xpowadd, onetile);
-                REL();
             }
         }
         // Sum(|x|^p)
-        ACQ();
+        tile_regs_acquire();
         cb_wait_front(cb_xpowadd, onetile);
         cb_reserve_back(cb_xpowsum, onetile);
 
-        reduce_init_delta<false>();
+        reduce_init_delta_with_dt<false>(cb_xpowsum, cb_xpowadd, cb_one);
         reduce_tile(cb_xpowadd, cb_one, 0, 0, dst0);
-        reduce_revert_delta();
+        reduce_revert_delta(cb_xpowsum);
+		tile_regs_commit();
 
-        pack_tile(dst0, cb_xpowsum);
+		tile_regs_wait();
+        pack_tile_with_dt(dst0, cb_xpowsum);
+		tile_regs_release();
```

Comment:
Why don't you re-format the code?

Reply:
I have applied formatting to the kernel code.